### ISSUE TITLE
Provide the same api as `config.plugins` for `config.optimization.minimizer`

### ISSUE THREAD

**Issue by Airkro** [neutrinojs/webpack-chain#95]
example: 

```js
config.optimization.minimizer.plugin(name).use(WebpackPlugin, args)
```

**Comment by edmorley** [comment]
Hi! Yeah I agree the API should match between the two. Though I suppose there's no way to avoid it being a breaking change?

If you'd like to tackle this, we'd gladly accept a PR :-)

**Comment by Airkro** [comment]
v5 is great, only one problem when I use `toString()`.

![image](https://user-images.githubusercontent.com/12206891/46641353-c92bd180-cba3-11e8-926e-6a0b99547b8d.png)

**Comment by eliperelman** [comment]
@Airkro what is the problem exactly?

**Comment by edmorley** [comment]
@eliperelman it took me a while to spot it too, but the `/* config.plugin('terser') */` should be `/* config.optimization.minimizer('terser') */` (guessing perhaps this issue applies to the resolve plugins too?).

**Comment by edmorley** [comment]
I fixed the output issue above in #116, which has just been released in 5.0.1.
